try {
	throw("test error");
	Sys.out.println("try");
} catch (string err) {
	Sys.out.println(err);
} finally {
	Sys.out.println("finaly");
}